lib/SPE.min.js

```diff
@@ -1,5 +1,5 @@
 /* shader-particle-engine 1.0.6
- * 
+ *
```

Comment:
How did you create SPE.min.js? You did the change directly in the file or you regenerated it in some way?

Reply:
Since our code currently has no way to regenerate SPE.min.js, I just monkey-patched it.  Unfortunately, https://github.com/squarefeet/ShaderParticleEngine has not been updated — there's no upstream update.

We could patch and rebuild the upstream repo, but as the author hasn't maintained it, we should probably just add to the webpack configuration to rebuild SPE.

Reply:
I don't think the SPE.min.js is actually used.
https://github.com/c-frame/aframe-particle-system-component/blob/master/index.js#L7C21-L7C32 is requiring SPE.js so it's included in the build and minified. SPE.min.js is not used, right? I think we can just remove it.

Reply:
Yes, it turns out webpack was already using SPE.js, so I've deleted SPE.min.js